src/action.js

```diff
@@ -37,6 +39,26 @@ async function exportSecrets() {
         }
     }
 
+    if (process.env['http_proxy'] || process.env['HTTP_PROXY']) {
```

Comment:
We should add this setting in the list of inputs supported by the action so it is explicit and documented.

Unless these environment variables have a special meaning and are required for the agent to work properly?

Reply:
I think it make sense since they are standard and not vault-specific. It's probably worth a mention in the README though, otherwise people would need to read the source code to know proxy servers are supported by the agent.